e2e/localbitsong/scripts/setup.sh

```
@@ -3,10 +3,16 @@
 CHAIN_ID=localbitsong
 BITSONG_HOME=$HOME/.bitsongd
 CONFIG_FOLDER=$BITSONG_HOME/config
-MONIKER=val
 STATE='false'
 
-MNEMONIC="bottom loan skill merry east cradle onion journey palm apology verb edit desert impose absurd oil bubble sweet glove shallow size build burst effort"
+KEY_PATH=$BITSONG_HOME/config/test-keys
```

Shellcheck (0.10.0)

[warning] KEY_PATH appears unused. Verify use (or export if used externally).

(SC2034)